.github/workflows/pnetcdf.yml

```diff
@@ -0,0 +1,113 @@
+            rm -f qout
+        - name: configure darshan-runtime and darshan-util
+          if: ${{ success() }}
+          run: |
+            cd ${GITHUB_WORKSPACE}
+            mkdir -p darshan_install
+            export DARSHAN_INSTALL_PATH=${GITHUB_WORKSPACE}/darshan_install
+            # clone autoperf module
+            git submodule update --init
+            ./prepare.sh
+            mkdir -p build
+            cd build
+            ../configure --prefix=$DARSHAN_INSTALL_PATH \
+                         --with-log-path-by-env=DARSHAN_LOGPATH \
+                         --with-jobid-env=NONE \
+                         --enable-pnetcdf-mod \
+                         --with-pnetcdf=${GITHUB_WORKSPACE}/PnetCDF \
+                         RUNTIME_CC=mpicc
+        - name: Dump log files if configure failed
+          if: ${{ failure() }}
+          run: |
+            cd ${GITHUB_WORKSPACE}/build
+            echo "-------- config.log --------"
+            cat config.log
+            echo "-------- darshan-runtime/config.log --------"
+            cat darshan-runtime/config.log
+            echo "-------- darshan-util/config.log --------"
+            cat darshan-util/config.log
+        - name: Test make, make check, and make install
+          if: ${{ success() }}
+          run: |
+            echo "make darshan-runtime and darshan-util"
+            cd ${GITHUB_WORKSPACE}/build
+            make -j 8
+            echo "-------- make check --------"
+            make check
+            echo "-------- make install --------"
+            make install
+        - name: Run PnetCDF make check
+          run: |
+            cd ${GITHUB_WORKSPACE}/PnetCDF/pnetcdf-${PNETCDF_VERSION}
+            make -s LIBTOOLFLAGS=--silent V=1 -j 8 tests > qout 2>&1
+            export LD_PRELOAD=${GITHUB_WORKSPACE}/darshan_install/lib/libdarshan.so
+            make check
+            unset LD_PRELOAD
+        - name: Print PnetCDF log files if make check failed
+          if: ${{ failure() }}
+          run: |
+            cd ${GITHUB_WORKSPACE}/PnetCDF/pnetcdf-${PNETCDF_VERSION}
+            cat src/utils/ncvalidator/*.log
+            cat test/*/*.log
+        - name: Run PnetCDF make ptest
+          run: |
+            cd ${GITHUB_WORKSPACE}/PnetCDF/pnetcdf-${PNETCDF_VERSION}
+            export LD_PRELOAD=${GITHUB_WORKSPACE}/darshan_install/lib/libdarshan.so
+            make ptest
+            unset LD_PRELOAD
+            make -s distclean >> qout 2>&1
+            rm -f qout
+        - name: Run Darshan make distcheck
+          if: ${{ success() }}
+          run: |
+            echo "make distcheck"
+            cd ${GITHUB_WORKSPACE}/build
+            make distcheck DISTCHECK_CONFIGURE_FLAGS="--with-log-path-by-env=DARSHAN_LOGPATH --with-jobid-env=NONE --enable-hdf5-mod --with-hdf5=${GITHUB_WORKSPACE}/HDF5 --enable-pnetcdf-mod --with-pnetcdf=${GITHUB_WORKSPACE}/PnetCDF RUNTIME_CC=mpicc"
```

Comment:
I'm guessing earlier CI failures are masking this, but this configure line is specifying to enable HDF5 module (along with an HDF5 install path), but I don't see any HDF5 installation as part of this workflow. Should we just remove this?